### ISSUE TITLE
Add recipes wildcard support

### ISSUE THREAD

**Issue by SSKirillSS** [codetaylor/pyrotech#179]
To make it easier to explain, simulate the situation:
With the Craft Tweaker was created a mass of recipes for a vanilla crafting bench, and you need to make sure that these same recipes are supported on the Pyrotech worktable. In a normal situation, you will have to get the names of all created recipes, and then add them to the worktable.
With a large number of recipes, this is very inconvenient. And it would be nice to add support for recipe names wildcard, like `crafttweaker:*`.

Maybe I explained a little (or a lot) incomprehensible, but I hope the opposite. In simpler terms - I want to quickly add recipes from the vanilla crafting bench to the pyrotech worktable.

**Comment by codetaylor** [comment]
Right, I'm pretty sure I know what you're talking about - I think maybe you mentioned it in Discord. 

Anyway, from what I understand, you're talking about the methods that take a resource location in string form like:
```java
static void blacklistVanillaRecipes(
  string[] resourceLocations
);
```
and
```java
static void whitelistVanillaRecipes(
  string[] resourceLocations
);
```

A resource location is `(domain):(path)`

So, supplying `(domain):*` would match any resource location with the given domain regardless of path.

**Comment by codetaylor** [comment]
Implemented in 1.4.0
